src/components/KnownForList.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <div
+        class="d-flex p-0 flex-row justify-content-center align-items-end container "
+    >
+      <div class="d-flex flex-row">
+        <b-list-group horizontal class="flex-sm-wrap">
+          <b-list-group-item
+              class="w-auto m-2 p-0 border-0 d-flex justify-content-center align-items-start rounded-circle card"
+              v-for="{
+              title,
+              backdrop_path,
+              overview,
+              original_language,
+              original_title,
+              release_date,
+              vote_average,
+              id,
+              media_type,
+            } in actorKnownFor"
+              :key="id"
+          >
+            <film-card
+                v-if="media_type === 'movie'"
+                :title="title"
+                :backdropPath="backdrop_path || ''"
+                :overview="overview"
+                :originalLanguage="original_language"
+                :originalTitle="original_title"
+                :releaseDate="release_date || ''"
+                :overallRating="vote_average"
+                :cardId="id"
+                @get-card-id="getChosenMovieDetails"
+                :known-for="true"
```

Comment:
Use more descriptive names for control props, i.e. "isKnownForListVisible" would probably be a better name here.